lib/radioactive_bunnies/worker.rb

```diff
@@ -73,6 +70,10 @@ def queue_name
       @queue_name
     end
 
+    def deadletter_exchange=(exchange)
+      @queue_opts[:deadletter_exchange] = exchange
```

Comment:
Well  ..... I was trying to use q_opts as the method arg and @queue_opts as the instance var.  Is there a place I mixed that up ?